grpc/build.gradle

```diff
@@ -0,0 +1,43 @@
+buildscript {
+    dependencies {
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.5'
```

Comment:
define dependency versions in versionsOfDependencies.gradle.

Reply:
Yes. 👍